internal/config/other.go

```diff
@@ -29,3 +29,10 @@ type RateLimits struct {
 	Rate    float64
 	Burst   int
 }
+
+type PosthogConfig struct {
+	Enabled     bool
+	ApiKey      string
```

Comment:
There is already a `RedactedString` type you can use for this in the gateway so that the api key does not show up in logs. It is already in `internal/config/redacted_string.go`. So it is already in the same package.